src/main/resources/product-service-openapi.yaml

```diff
@@ -0,0 +1,341 @@
+---
```

Comment:
Can we have comments in YAML?
Then we should add a comment and link to tell where this is comming from.